bitcoin/src/sign_message.rs

```diff
@@ -126,6 +125,14 @@ mod message_signing {
             })
         }
 
+        /// Create from a byte slice.
```

Comment:
nit, this could be improved to be similar to the docs on `from_byte_array`.

Reply:
Done